CONTRIBUTING.md

```diff
@@ -169,6 +171,10 @@ When declaring a record in XML,
 </record>
 ```
 
+### Records
+
+* For records of model `ir.filters` use explicit `user_id` field.
```

Comment:
Thank for comment.
You are right!
I will add a example and reference to issue.